wazimap_ng/utils.py

```diff
@@ -615,3 +620,41 @@ def clean_columns(file):
     new_columns = df.columns.str.lower().str.strip()
     file.seek(0)
     return old_columns, new_columns
+
+
+def error_handler(request, *args, **kwargs):
+
+    message = 'Server Error occurred.'
+    error_type = 'server_error'
+    code = status.HTTP_500_INTERNAL_SERVER_ERROR
+
+    data = {
+        'error': {
+            'message': {'detail': message},
+            'type': error_type,
+            'code': code
+        }
+    }
+
+    return JsonResponse(
+        data, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+def custom_exception_handler(exc, context):
+    from rest_framework.views import exception_handler
```

Comment:
why did you put the import here and not at the top with the other imports?

Reply:
It results in a circular import error.